src/fluids/neutral/initneutral.F90

```diff
@@ -248,11 +248,11 @@ subroutine flux_neu(this, flux, cfr, uu, n, vx, ps, bb, cs_iso2)
       real, dimension(:,:), intent(inout), pointer :: cfr       !< freezing speed for neutral fluid
       real, dimension(:,:), intent(in),    pointer :: uu        !< part of u for neutral fluid
       real, dimension(:),   intent(in),    pointer :: vx        !< velocity of neutral fluid for current sweep
-      real, dimension(:),   intent(inout), pointer :: ps        !< pressure of neutral fluid for current sweep
       real, dimension(:,:), intent(in),    pointer :: bb        !< magnetic field x,y,z-components table
       real, dimension(:),   intent(in),    pointer :: cs_iso2   !< isothermal sound speed squared
 
       ! locals
+      real, dimension(:), pointer :: ps        !< pressure of neutral fluid for current sweep
```

Comment:
This needs to be allocated. And deallocated to avoid leaks. Note that usually same vector length is used few times ina row each time, so only some allocate/deallocate are necessary. Same in other fluids.